data/fundingProgrammes.js

```diff
@@ -0,0 +1,556 @@
+'use strict';
+module.exports = {
```

Comment:
🤔 Interesting. I was considering we might tag the programmes in the CMS by code (e.g.. AFA1, AFA2) etc.

I think we're better off considering this an enhancement for current programmes, rather than trying to provide a mapping for everything.

Reply:
Thinking about this, not sure my suggestion is better. This is the same trade-off in the other direction, so lets go with this.

Reply:
Assembling that list (eg. a probably-slow lookup) where we only have about 30 of the 100+ programmes in the database means we'd be unable to provide even a basic URL for most cases, this way means there's a nice minimum. It'll all probably change when the data warehouse is ready anyway, this just means we can do the basic thing for now.

Reply:
Ah great shout, I didn't notice you'd be able to link out to archived programmes this way. Nice work!